### DISCUSSION TITLE
关于刷题频率

### DISCUSSION THREAD

**Discussion by asdf2014** [asdf2014/algorithm#89] [General]
目前是[每周一题](https://github.com/asdf2014/algorithm#刷题频率)，但是只是一个指导性计划。也可以有自己的刷题频率，以及选题模式。大家一般会采用什么刷题模式呢，欢迎留言讨论 😄

**Comment by leiwingqueen** [comment]
leetcode现在每个月的专题形式其实就挺好的。一个月一个TAG，一个月能熟练掌握一个tag其实就够了
